.buildkite/scripts/post-release.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common.sh
+
+TAG="v{$1}"
+
+set_git_config() {
+    git config user.name "${GITHUB_USERNAME_SECRET}"
+    git config user.email "${GITHUB_EMAIL_SECRET}"
+}
+
+tag_commit() {
+  echo "Tagging commit ${BUILDKITE_COMMIT}"
+  git tag -a -m "${BUILDKITE_COMMIT}" "${TAG}"
+}
+
+git_push_with_auth() {
+  echo "Pushing tag ${TAG}"
+  retry 3 git push https://${GITHUB_USERNAME_SECRET}:${GITHUB_TOKEN_SECRET}@github.com/elastic/golang-crossbuild.git ${TAG}
```

Comment:
This command does not look like exactly the same as in Jenkins:
https://github.com/elastic/golang-crossbuild/blob/b66e5a57400436f8fb206ad2742a394ffb9d0104/Jenkinsfile#L249C7-L249C19

In jenkins it is used `--force` parameter for pushing the tag.

https://github.com/elastic/apm-pipeline-library/blob/5800f4a0e2c6b29ff642b29279cb4b6eb9095cbc/vars/gitCreateTag.groovy#L31
https://github.com/elastic/apm-pipeline-library/blob/5800f4a0e2c6b29ff642b29279cb4b6eb9095cbc/vars/gitPush.groovy#L28

Should it be something like this ?
```suggestion
  retry 3 git push https://${GITHUB_USERNAME_SECRET}:${GITHUB_TOKEN_SECRET}@github.com/elastic/golang-crossbuild.git --tags --force
```

Reply:
I was suspicious about force pushing tags, since that might lead to issues like conflicts between remote-local tags etc. And ideally tag always should point in a same commit and should not be moved to avoid confusion. I understand that there is additional check in post-release script, but anyways I'd try to avoid force pushing.

Reply:
Confirmed with  @jlind23 that force pushing is not needed anymore.